scripts/reprocess_quarantine.py

```diff
@@ -2,32 +2,36 @@
 from pika import URLParameters
 from sdc.rabbit.publishers import QueuePublisher, PublishMessageError
 
-from app import create_and_wrap_logger
-from app import settings
-
-logger = create_and_wrap_logger(__name__)
-
 
 def reprocess():
-    connection = BlockingConnection(URLParameters(settings.RABBIT_URL))
+    # To use this script, fill in the credentials for rabbit.  They should be found in app/settings.py
+    rabbit_url = 'amqp://<user>:<pass>@<host>:<port>/%2f'
+    connection = BlockingConnection(URLParameters(rabbit_url))
     channel = connection.channel()
-    method, properties, body = channel.basic_get(settings.RABBIT_QUARANTINE_QUEUE)
+    method, properties, body = channel.basic_get('Seft.Responses.Quarantine')
     if method:
         try:
-            logger.info("Recovered quarantine message", body=body, headers=properties.headers)
-            publisher = QueuePublisher(urls=settings.RABBIT_URLS,
-                                       queue=settings.RABBIT_QUEUE)
+            print("Recovered quarantine message")
```

Comment:
Why has it gone from using a logger to using print?

Reply:
The problem this is solving is that running the script complains that it can't see 'app'.  Even if we did solve this problem by copying it from /scripts to /app/scripts, the next problem is that the settings look at environment variables for their values and they're not available at the time you're running the script (without then running other stuff to get it into the environment)

Reply:
There's probably a better solution to this, but it happens so infrequently that spending a minute putting in 4 bits of information to run the script is good enough.